src/main/jte/index.jte

```diff
@@ -24,12 +24,12 @@
                     return event.display === 'background';
                 },
                 selectAllow: function(selectInfo) {
-                    return selectInfo.start < new Date(new Date().getTime() + 15 * 60 * 1000)
+                    return selectInfo.start > new Date(new Date().getTime() + 15 * 60 * 1000)
```

Comment:
This fixes FullCalendar time range selection, which is required for the drag-based event creation